### PR TITLE
Add projects questions

### PR DESCRIPTION
**Pull request by Wardormeur**
Add redirection to na-2018 by default
Fix some quick display issue based on the event info (landing page)

### CODE REVIEW

File: client/src/event/EventUtilsMixin.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import moment from 'moment';
+
+export default {
+  computed: {
+    formattedDate() {
+      return moment(this.event.date).format('LLLL');
```
[the remark this answers is not in the document]
Comment (DanielBrierton) [reply client/src/event/EventUtilsMixin.js]: I'm wondering if we should actually store the timezone of the event, and store the time of the event in UTC, then do `moment(this.event.date).tz(this.event.tz).format('LLLL zz')` to print the time in the correct timezone along with what the timezone is.

I feel like this is important for any event that spans multiple timezones, like Coolest Projects NA.

[the remark this answers is not in the document]
Comment (DanielBrierton) [reply client/src/event/EventUtilsMixin.js]: OR...

**Comment by Wardormeur** [comment]
Needs splitting into master/usa

**Comment by Wardormeur** [comment]
Replaced by https://github.com/CoderDojo/coolest-platform/pull/41 and https://github.com/CoderDojo/coolest-platform/pull/42

**Pull request closed without merging**